apps/snitch_core/priv/repo/seed/variation_theme.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Snitch.Seed.VariationTheme do
+
+  alias Snitch.Core.Tools.MultiTenancy.Repo
+  alias Snitch.Data.Schema.VariationTheme
+  alias Snitch.Data.Schema.OptionType
+
+  defp get_option_type do
+    Repo.all(OptionType)
+  end
+
+  def seed!() do
+    option_types = get_option_type
+    option_ids = for option <- option_types, do: to_string(option.id)
+    option_names = for option <- option_types, do: option.name
+    for option <- option_types, do: create_variation_theme(option.name, [to_string(option.id)])
+    create_variation_theme(Enum.join(option_names, "-"), option_ids)
```

Comment:
@jyotigautam Not the best logic but will do for now. Just think about it, if we decide to have let's say 3 option types then this would fail badly.

Reply:
Yes, it would not create all the possible combinations of option-types for a variation theme, hence we can use combination library to create all the combinations.